Partial revert of
https://github.com/snyk/cli/commit/af678f3d6aadc5969d64aea20bfbca83c980fccf to
avoid using the _next_ version as the version for our current build.
---
diff --git a/release-scripts/next-version.sh b/release-scripts/next-version.sh
index 05e3bbb..b23b8fd 100755
--- a/release-scripts/next-version.sh
+++ b/release-scripts/next-version.sh
@@ -3,7 +3,7 @@ set -euo pipefail
 # Checks the latest version of Snyk CLI on npm and decides the next version.
 # Only output the next version to stdout. All other output should go to stderr.
 
-NEXT_VERSION="$(convco version --bump --patch)"
+NEXT_VERSION="$(convco version --bump)"
 CURRENT_TAG="$(git describe --tags `git rev-list --tags --max-count=1`)"
 RELEASE_CHANNEL="$($(dirname "$0")/determine-release-channel.sh)"
